### DISCUSSION TITLE
Feature idea: Netlify build plugin

### DISCUSSION THREAD

**Discussion by themihel** [lycheeverse/lychee#861] [Ideas]
Hey folks,

just an idea you may, or may not, follow: How about supporting Lychee as Netlify build plugin? (relates partially to #59)

This may lead to some positive side effects:
* An increased visibility for the project itself (displayed in their plugin directory)
* Integrates easily into many peoples build processes (including myself) :)

Further information can be found here:
* https://www.netlify.com/products/build/plugins/
* https://docs.netlify.com/configure-builds/build-plugins/create-plugins/

Free "market research":
Just found one link checker already on the platform "checklinks" [1]. A more diverse selection is of course always nice to have :)

[1] https://github.com/munter/netlify-plugin-checklinks#readme

**Comment by mre** [comment]
I really like that idea! As discussed via chat, we can build packages for Node and Python.
For Node we can look into [Neon](https://github.com/neon-bindings/neon) and for Python [pyo3](https://github.com/PyO3/pyo3). At least the Node/npm package would be a requirement for the Netlify plugin because they only support JS/Typescript packages as plugins at the moment.

**Comment by mre** [comment]
Moved this to a discussion as there's currently little bandwidth to work on additional language support. I do hope we get to that after 1.0, though.
